.github/workflows/checks.yml

```diff
@@ -35,38 +35,9 @@ jobs:
       run: mypy --python-version ${{ matrix.python }} -p build
 
 
-  flake8:
+  pre-commit: # per https://github.com/pre-commit/action
```

Comment:
I do not think this comment is needed.

Reply:
I think it helps with where this setup recommendation comes from 🤔

Reply:
The recommendation is a simple example of how to use a github action, I don't think it adds any extra value.